cpp/examples/jsonschema/writer.cpp

```diff
@@ -0,0 +1,98 @@
+// Example of writing JSON messages to an MCAP file in C++.
+// Writes a topic of Point2 messages.
+#define MCAP_IMPLEMENTATION
+#include "mcap/writer.hpp"
+
+#include <nlohmann/json.hpp>
+
+#include <chrono>
+#include <cmath>
+#include <iostream>
+
+#define NUM_FRAMES 100
+#define NS_PER_MS 1000000
+
+static const char* SCHEMA_NAME = "foxglove.Point2";
+static const char* SCHEMA_TEXT =
+  "{ \
+  \"$comment\": \"Generated from Point2 by @foxglove/schemas\", \
+  \"title\": \"Point2\", \
+  \"description\": \"A point representing a position in 2D space\", \
+  \"type\": \"object\", \
+  \"properties\": { \
+    \"x\": { \
+      \"type\": \"number\", \
+      \"description\": \"x coordinate position\" \
+    }, \
+    \"y\": { \
+      \"type\": \"number\", \
+      \"description\": \"y coordinate position\" \
+    } \
+  } \
+}";
+
+int main(int argc, char** argv) {
+  if (argc != 2) {
+    std::cerr << "Usage: " << argv[0] << " <output.mcap>" << std::endl;
+    return 1;
+  }
+  const char* outputFilename = argv[1];
+
+  mcap::McapWriter writer;
+  {
+    auto options = mcap::McapWriterOptions("x-jsonschema");
+    const auto res = writer.open(outputFilename, options);
+    if (!res.ok()) {
+      std::cerr << "Failed to open " << outputFilename << " for writing: " << res.message
+                << std::endl;
+      return 1;
+    }
+  }
+  // Create a channel and schema for our messages.
+  // A message's channel informs the reader on the topic those messages were published on.
+  // A channel's schema informs the reader on how to interpret the messages' content.
```

Comment:
```suggestion
  // A channel's schema informs the reader of how to interpret the messages' content.
```
or maybe this is just an AU/US difference I wasn't aware of?